bears/general/LicenseCheckBear.py

```diff
@@ -0,0 +1,40 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='licensecheck',
+        use_stdout=True,
+        output_format='regex',
+        output_regex=r'.*: .*UNKNOWN',
+        result_message='No license found.')
+class LicenseCheckBear:
+    """
+    Attempts to check the given file for a license, by searching the start
+    of the file for text belonging to various licenses.
+    """
+    LANGUAGES = {'All'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='devscripts',
+                                            dnf='licensecheck')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'License'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         licensecheck_lines: int=None,
+                         licensecheck_tail: int=None):
+        """
+        :param licensecheck_lines:
+            Specify how many lines of the file header should be parsed
+            for license information.(Default: 60)
```

Comment:
> We shouldn't merge anything until it is perfect.

I'm also for this attitude, but sometimes you can drive this too much ;) Better get something reasonable than nothing at all. Our PRs do already require a lot of time because of our high standard.

not everyone has support for .deb, I use fedora, so default is .rpm

Maybe you can assert the program version and change behavior depending on that? `check_prerequisites` could do the trick^^